tests/libtransmission/json-test.cc

```diff
@@ -24,27 +25,18 @@ class JSONTest : public ::testing::TestWithParam<char const*>
 protected:
     void SetUp() override
     {
+        ::testing::TestWithParam<char const*>::SetUp();
+
         auto const* locale_str = GetParam();
         try
         {
-            old_locale_ = std::locale::global(std::locale{ {}, new std::numpunct_byname<char>{ locale_str } });
+            tr_locale_set_global(locale_str);
```

Comment:
Not sure why these changes in `SetUp()` and `TearDown()` were necessary, but I'd start from rolling them back and seeing if it helps.

Reply:
I see, the util function does a bit more than changing numpunct locale. Still it'd be nice to restore the old locale in `TearDown()` so that the other tests (and apparently GTest) aren't affected.

Reply:
041faba Thanks, finally got the CI green.

Reply:
@mikedld thank you, nice work!